orc8r/cloud/go/services/accessd/storage/storage_blobstore.go

```diff
@@ -67,20 +69,20 @@ func (a *accessdBlobstore) ListAllIdentity() ([]*protos.Identity, error) {
 		return nil, status.Errorf(codes.Internal, "failed to get many acls: %s", err)
 	}
 
-	ret := make([]*protos.Identity, 0, len(tks))
+	ids = make([]*protos.Identity, 0, len(tks))
```

Comment:
This line is no longer necessary

Reply:
@hcgatewood Isn't a one time allocation of memory for the "ids" more efficient than leaving it up to append() to constantly reallocate and move the contents of the array? Because if we took away the make(), wouldn't the capacity of ids be 0 which means that every append would allocate (1, ..., n) times whereas make() would only do it once.

Reply:
Two things:

- you're right that the underlying array is reallocated successively as its capacity becomes insufficient, but the runtime is smart enough to ~double the capacity of the array, meaning you only need log n reallocations
- for code that's non-performance-critical, default to the nil slice + append idiom for readability. For performance-critical code, e.g. FeG code and certain per-subscriber functionalities in orc8r, prefer denoting explicit capacity via `make`

In this case, this code is not performance critical, since (a) it's only ever called from the `accessc` command and (b) the full set of operators is expected to remain relatively small. If we were listing the full set of subscribers, e.g., I would say use `make` though

All things considered, my comment is more of a nit. But I'm glad you've put thought into whether the line should be present. I'll leave the final decision up to you